tests/webtests/CreateConnectionRequestTest.php

```diff
@@ -61,15 +62,20 @@ public function test_a_connection_request_can_be_created()
         self::assertSelectorIsDisabled('#connection_request_container_send');
         self::assertSelectorIsEnabled('#connection_request_container_cancel');
 
+        /** @var TranslatorInterface $translator */
+        $translator = self::getContainer()->get(TranslatorInterface::class);
+
+        self::assertSelectorTextContains('.empty-connection-list-message', $translator->trans('entity.create_connection_request.emptyConnectionListPlaceholder'));
         $this->fillFormField($form, '#connection_request_container_connectionRequests___name___institution', 'Institution 1');
         $this->fillFormField($form, '#connection_request_container_connectionRequests___name___name', 'Jesse');
         $this->fillFormField($form, '#connection_request_container_connectionRequests___name___email', 'jesse-james@gmail.com');
         $this->click($form, '.add_collection_entry');
 
         self::assertSelectorExists('.collection-list');
         self::assertSelectorIsVisible('.collection-list');
+        self::assertSelectorTextNotContains('.empty-connection-list-message', 'No items have been added yet.');
 
-        self::assertSelectorTextContains('.collection-list td:nth-child(1)', 'Institution 1');
+        self::assertSelectorTextContains('.collection-list tr.collection-entry td:nth-child(1)', 'Institution 1');
```

Comment:
No option to use the translator here? Feel free to ignore/resolve if you find this feedback insignificant.